lib/stdlib/src/array.erl

```diff
@@ -1,8 +1,3 @@
-%%
```

Comment:
I don't agree that the Ericsson Copyright should be removed here. It is Dan Gudmundsson who has done the major job on this module as an Ericsson employee. We should keep the Ericsson Copyright and possibly change order of the authors.
Please change this. Otherwise I think the rest of the pull request is ok.